packages/api/amplify_api/example/integration_test/util.dart

```diff
@@ -35,14 +44,15 @@ class TestUser {
   final String _username;
   final String _password;
 
-  Future<void> signUp() async {
+  String get testEmail => '$_username@amazon.com';
```

Comment:
*suggestion:* I would add a uuid even if it isn't required right now. If this user isn't deleted after the test run and a subsequent run does not handle the case where the user already exists that may be an issue.